it/v1/endpoints/ListBenefitsControllerISpec.scala

```diff
@@ -57,32 +57,60 @@ class ListBenefitsControllerISpec extends IntegrationBaseSpec {
 
   }
 
+  private trait NonTysTest extends Test {
+    def uri: String = s"/$nino/$taxYear"
+
+    def downstreamUri: String = s"/income-tax/income/state-benefits/$nino/$taxYear"
+  }
+
+  private trait TysIfsTest extends Test {
+    def uri: String = s"/$nino/$taxYear"
```

Comment:
Might be renaming `uri` to `mtdUri` just for clarity. Also, it can be defined once in test rather than in NonTysTest and TysIfsTest as its got the same value in both